### PR TITLE
Dockerfile with dataset and other dependencies

### PR DESCRIPTION
**Pull request by acheshkov**
Two docker images:

1. image contains dataset and other artifacts Aibolit need
2. image prepares an environment to train a model

### CODE REVIEW

**Comment by acheshkov** [comment]
closes #256

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/257#issuecomment-621970208))

@acheshkov Job `gh:yegor256/aibolit#257` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/257#issuecomment-621970208))

@acheshkov Pull request [#257](https://github.com/yegor256/aibolit/issues/257) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: docker/Dockerfile.download [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM ubuntu
+ADD https://github.com/yegor256/aibolit/releases/download/v1.0.0/dataset.zip .
+ADD https://github.com/yegor256/aibolit/releases/download/v1.0.0/halstead.jar .
```
Comment (yegor256) [inline docker/Dockerfile.download:3]: @acheshkov if you do this:

```
RUN wget https://github.com/yegor256/aibolit/releases/download/v1.0.0/halstead.jar
ADD halstead.jar
```

The problem will be solved, I believe.

Comment (acheshkov) [reply docker/Dockerfile.download]: Thanks, done

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#257` is not assigned, can't get performer

**Pull request merged**